tsconfig.json

```diff
@@ -5,6 +5,7 @@
   "compilerOptions": {
     "outDir": "dist-types",
     "incremental": false,
-    "sourceMap": true
+    "sourceMap": true,
+    "jsx": "react",
```

Comment:
do we want to keep this? i had to add this to get warnings in webstorm to go away

Reply:
Huh; I expected that to come from https://github.com/cortexapps/backstage-plugin/blob/e68724afe8ce9a56380f01d6d715d2aa55cddabc/tsconfig.json#L2

but apparently they don't provide that, so your change looks correct
https://github.com/BackstageJS/backstage-cli/blob/534e1370bfbc2b47f7e2731e2b5164be14106f70/tsconfig.json#L9